lib/show_n_tell/github/authenticator.ex

```diff
@@ -24,6 +24,7 @@ defmodule ShowNTell.Github.Authenticator do
     {:fail, body}
   end
 
+  defp check_github_org({:ok, %{status: 401} = resp}), do: {:fail, %{"error" => Map.get(resp.body, "message")}}
```

Comment:
Cool - this handles the 401 - unauthorized, un addition to the "catch-all" error handler below.